app/views/govuk_publishing_components/components/_image_card.html.erb

```diff
@@ -23,10 +23,13 @@
   ]
   extra_link_classes << brand_helper.color_class
 
+  data_modules = ""
+  data_modules << " gem-track-click" if card_helper.is_tracking?
+  data_modules << " image-card" if defined?(youtube_video_id)
```

Comment:
Non-blocking: I think you can use `%w[ ]` in order to not have to worry about leading whitespace, e.g.

```
data_modules = %w[]
data_modules << "gem-track-click" if card_helper.is_tracking?
data_modules << "image-card" if defined?(youtube_video_id)
```

Reply:
Have rewritten this div to be a content_tag so we can just use `%w[]` for all the conditionally added attributes/classes.